engine/difficulty.cpp

```diff
@@ -0,0 +1,35 @@
+#include "difficulty.h"
+
+namespace pong
+{
+
+Difficulty::Level Difficulty::sLevel = Level::Normal;
+
+Difficulty::Level Difficulty::GetLevel()
+{
+    return sLevel;
+}
+
+void Difficulty::SetLevel(Level level)
+{
+    sLevel = level;
+}
+
+std::string DifficultyString(Difficulty::Level level)
```

Comment:
typically this method would be named "to_string", it's a special thing that other classes have as well, as an experienced C++ dev, it'd be one of the first things I'd try on a new class

Reply:
before you ask: yes C++ has messed up conflicts of style, snake_case vs CamelCase...